README.md

```diff
@@ -1 +1,25 @@
-# Portolio_mobile_view
\ No newline at end of file
+# Portfolio_integration 
+ 
+My personal Portfolio that gives a clear over view of who I am. It hightlights my skills and links on how to get to me on different platforms  
+
+# Build with 
+ - HTML 
+ - CSS
+
+
```

Comment:
- [ ] [OPTIONAL] To make your documentation professional, you can consider adding live demo link of you portfolio.